.github/workflows/pr-checks.yml

```diff
@@ -7,7 +7,7 @@ on:
   pull_request:
```

Comment:
By the way, it should be easy enough to avoid running these tests if only markdown files are changed.

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-excluding-paths

Reply:
Thanks, done